OpenRA.Mods.Common/Traits/Demolishable.cs

```diff
@@ -90,10 +92,13 @@ void ITick.Tick(Actor self)
 			}
 
 			// Remove expired actions to avoid double-revoking
-			foreach (var a in removeActions)
-				actions.Remove(a);
+			if (removeActions.Count > 0)
```

Comment:
Is this necessary? I don't think an empty foreach and clear cause any perf issues